src/main/java/com/nexenio/CppModernJson/CppModernJsonCodegen.java

```diff
@@ -82,6 +82,7 @@ public String getHelp() {
 
   public CppModernJsonCodegen() {
     super();
+    supportsInheritance = true;
```

Comment:
Do we need to double check if this is correct?
Is it needed? Is this how and where it should be set?

Reply:
When leaving this out, superclass members would be flattened into the subclasses. That is not what we want.
Also, this is the same way and place where other code generators set this property, so I'm fairly sure that this is where and how it should be.